org.alloytools.alloy.application/src/main/java/se/src/alloy/cli/Printer.java

```diff
@@ -0,0 +1,40 @@
+package se.src.alloy.cli;
+
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+
+public class Printer {
+
+  public static final String COLOR_RESET = "\033[0m";
```

Comment:
These ANSI codes do not work on all terminals. Not sure if we should use it because if it is not supported it looks awful.

Reply:
Agreed.  But if you figure out a way to check whether the terminal supports colors and using them only if it does, that would be awesome